pkgs/development/tools/gops/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "gops";
+  version = "0.3.15";
+
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = "gops";
+    rev = "v${version}";
+    sha256 = "091idnsgbwabmm5s9zhm474fbxvjvpkvwg68snbypfll7wdr3phy";
+  };
+
+  vendorSha256 = null;
+
+  # Test are disabled as they perform 'mkdir' in the HOME directory.
+  doCheck = false;
```

Comment:
`preCheck = "export HOME=$(mktemp -d)";` could do the trick to run the tests.

Reply:
Yes, indeed, setting `HOME` to tmp dir helps. Thanks for the suggestion.